packages/core/test/Form_test.js

```diff
@@ -2245,6 +2245,26 @@ describeRepeated("Form common", createFormComponent => {
     });
   });
 
+  describe("html5Validate", () => {
+    it("should remove noValidate when html5Validate is true", () => {
```

Comment:
Let's also add a test for when html5Validate is false.

Reply:
added a test that checks for both explicitly passing false and not passing anything